pyvrp/cli.py

```diff
@@ -310,6 +254,9 @@ def main():
     msg = "Whether to scale stopping criteria values by the number of clients."
     stop.add_argument("--per_client", action="store_true")
 
+    msg = "Whether to display information about the solver progress."
+    parser.add_argument("--display", action="store_true", help=msg)
```

Comment:
Does this work without passing the `stats_dir` argument as well? And how does this interact with the progress bar when solving a set of instances? I am unsure if we should support this from the CLI.

If we do, the argument should probably be added above the `stop` argument group. It's much easier to separate the different parts of the CLI that way.

Reply:
I find it very useful to have `display` available in the CLI when e.g. solving new VRP variants and see if the search is progressing or not. 

I have added a warning about this when solving a set of instances in parallel. When it's not in parallel, then the output is still OK in the sense that the progress bar doesn't update in place. Does that work for you?